### PR TITLE
bluetooth: controller: Refactor PDU_RX_POOL_SIZE

### PR DESCRIPTION
**Pull request by mped-oticon**
Two commits:

1. Refactor PDU_RX_POOL_SIZE into its constituents and name each term.
2. Remove PDU_RX_SIZE_MIN. See commit message for RX_CNT+1 explanation.

Signed-off-by: Mark Ruvald Pedersen <mped@oticon.com>

### CODE REVIEW

File: subsys/bluetooth/controller/ll_sw/ull.c [hunk 1]
```diff
@@ -137,21 +137,26 @@ static struct {
...
+#define PDU_RX_NODE_POOL_ELEMENT_SIZE                      \
+	MROUND(                                            \
+		NODE_RX_STRUCT_OVERHEAD                    \
+		+ MAX(PDU_ADVERTIZE_SIZE, PDU_DATA_SIZE)   \
+	)
+
+#define PDU_RX_POOL_SIZE (PDU_RX_NODE_POOL_ELEMENT_SIZE * (RX_CNT + 1))
 
 static struct {
 	u16_t size; /* Runtime (re)sized info */
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/ull.c]: Remove this member?

**Review by cvinayak — CHANGES_REQUESTED** [review]
Minor CR, otherwise LGTM

File: subsys/bluetooth/controller/ll_sw/ull.c (hunk 1 above)
Comment (mped-oticon) [reply subsys/bluetooth/controller/ll_sw/ull.c]: Fixed.

**Comment by mped-oticon** [comment]
> Minor CR, otherwise LGTM

**Comment by mped-oticon** [comment]
@cvinayak please recheck

**Review by cvinayak — APPROVED** [review]

**Review by thoh-ot — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**